### DISCUSSION TITLE
[Projects beta]: Hide status on board view

### DISCUSSION THREAD

**Discussion by ronaldarndt** [community/community#7242] [Projects and Issues] [answered]
I would like to hide a specific status from the board view so i can organize better.

**Comment by redcatbear** [comment]
I second that idea, because then you can separate partial boards by responsibilities. 
E.g. you have one for issue grooming, where someone grooms the tickets and moves them to the appropriate state. And after that developers take over and work on the tickets and see only the states they are interested in and responsible for.

**Comment by LivioMirandaJr** [comment]
+1

**Comment by rileybroughten** [comment, marked as answer]
Hello! 👋 You can select a column to hide from the `...` board column menu, and toggle which are visible from the `+` button to the right of your columns!

https://user-images.githubusercontent.com/101840513/225456101-928e00b4-aa87-4d3e-9b60-6b68ce563588.mov
